BurnOutSharp/ProtectionType/RingPROTECH.cs

```diff
@@ -37,7 +37,7 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
         {
             var matchers = new List<PathMatchSet>
             {
-                new PathMatchSet(new PathMatch("protect.pro", useEndsWith: true), "Ring PROTECH / ProRing [Check disc for physical ring]"),
+                new PathMatchSet(new PathMatch("protect.pro", useEndsWith: true), "Ring PROTECH / ProRing [Check disc for physical ring] (Unconfirmed - Please report to us on Github)"),
```

Comment:
Please double check because I think this was already a confirmed path.

Reply:
I wasn't able to confirm this for myself as of yet, but I can try to look further.